plugins/woocommerce-admin/client/guided-tours/wc-addons-tour/index.tsx

```diff
@@ -0,0 +1,81 @@
+/**
+ * External dependencies
+ */
+import { useEffect, useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { TourKit, TourKitTypes } from '@woocommerce/components';
+import qs from 'qs';
+
+/**
+ * Internal dependencies
+ */
+import { waitUntilElementTopNotChange } from '../add-product-tour/utils';
+import { getTourConfig } from './get-config';
+
+const WCAddonsTour = () => {
+	const [ showTour, setShowTour ] = useState< boolean >( false );
+	const [ tourConfig, setTourConfig ] = useState< TourKitTypes.WooConfig >();
+
+	const closeHandler: TourKitTypes.CloseHandler = (
+		steps,
+		currentStepIndex
+	) => {
+		setShowTour( false );
+		if ( steps.length - 1 === currentStepIndex ) {
+			// TODO: Track Tour as completed
+		} else {
+			// TODO: Track Tour as dismissed
+		}
+	};
+
+	const onNextStepHandler = ( stepIndex: number ) => {
+		// const stepName = tourConfig?.steps[ stepIndex ]?.meta?.name || '';
+		// TODO: Maybe scroll to a proper element
+		// TODO: Track tour's step as completed
+	};
+
+	const onPreviousStepHandler = ( stepIndex: number ) => {
+		// TODO: Maybe scroll to a proper element
+	};
+
+	const defaultAutoScrollBlock: ScrollLogicalPosition = 'center';
+
+	useEffect( () => {
+		const theConfig = getTourConfig( {
+			closeHandler,
+			onNextStepHandler,
+			onPreviousStepHandler,
+			autoScrollBlock: defaultAutoScrollBlock,
+		} );
+
+		setTourConfig( theConfig );
+
+		const query = qs.parse( window.location.search.slice( 1 ) );
+		if ( query && query.tutorial === 'true' ) {
```

Comment:
this could be simplified a bit
```suggestion
		if ( query?.tutorial === 'true' ) {
```

Reply:
Indeed, fixed in https://github.com/woocommerce/woocommerce/pull/35268/commits/2e117d9aaea033096a0759a0b987e1251b3b1e7e.